Xamarin.Forms.Platform.Android/CollectionView/ItemContentView.cs

```diff
@@ -7,6 +7,7 @@ namespace Xamarin.Forms.Platform.Android
 	public class ItemContentView : ViewGroup
 	{
 		protected IVisualElementRenderer Content;
+		internal Element Element => Content?.Element;
```

Comment:
What is this for?

Reply:
So i can access the Element and check it's position on the ItemsSource using **index = ItemsViewAdapter.GetPositionForItem(bContext);**